fe/fe-core/src/main/java/com/starrocks/common/util/Util.java

```diff
@@ -462,4 +465,12 @@ public static ConnectContext getOrCreateConnectContext() {
         ctx.setGlobalStateMgr(GlobalStateMgr.getCurrentState());
         return ctx;
     }
+
+    public static String hideToken(String url) {
```

Comment:
`WebUtils.sanitizeHttpReqUri` does exactly the same thing, can use it here

Reply:
> `WebUtils.sanitizeHttpReqUri` does exactly the same thing, can use it here

I see what you mean. Someone has taken care of these security issues, but they've left out these places. I'll reuse the previous code.